test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -2576,7 +2576,7 @@ class RepresenteeControllerSpec
       val allQuestionsAnswers = IncompleteRepresenteeAnswers(
         Some(completeAnswers.name),
         Some(completeAnswers.id),
-        completeAnswers.dateOfDeath,
+        completeAnswers.dateOfDeath.fold[Option[DateOfDeath]](Some(sample[DateOfDeath]))(dod => Some(dod)),
```

Comment:
it would be neater to populate a concrete date of death above to have available to reference like this:
```scala
val dateOfDeath = sample[DateOfDeath]
val completeAnswers = sample[CompleteAnswers].copy(dateOfDeath = Some(dateOfDeath))
val allQuestionAnswered = IncompleteAnswers(..., Some(dateOfDeath), ...)
```